src/types/onyx/Policy.ts

```diff
@@ -31,6 +31,18 @@ type DisabledFields = {
     reimbursable?: boolean;
 };
 
+// These types are for the Integration connections for a policy (eg. Quickbooks, Xero, etc).
+// This data is not yet used in the codebase which is why it is given a very generic type, but the data is being put into Onyx for future use.
+// Once the data is being used, these types should be defined appropriately.
+type ConnectionLastSync = Record<string, unknown>;
+type ConnectionData = Record<string, unknown>;
+type ConnectionConfig = Record<string, unknown>;
+type Connection = {
+    lastSync?: ConnectionLastSync;
+    data: ConnectionData;
+    config: ConnectionConfig;
```

Comment:
These will end up being quite different from integration to integration.

Reply:
Cool, thanks for some context about that! Sounds like leaving it generic for now is a good approach.